ee/app/canned-responses/server/settings.js

```diff
@@ -1,8 +1,8 @@
 import { settings } from '../../../../app/settings';
 
 export const createSettings = () => {
-	settings.addGroup('Canned Responses', function() {
-		this.section('Canned Responses', function() {
+	settings.addGroup('Canned_Responses', function() {
```

Comment:
Hey @ggazzo , this is the line that has introduced the duplication.

Reply:
Which duplication? 👀

Reply:
The one we're fixing in #23056

Reply:
Ohhh, noticed it 😬 got it.